src/gdk-pixbuf-1-fixes.patch

```diff
@@ -3,85 +3,32 @@ This file is part of MXE. See LICENSE.md for licensing information.
 Contains ad hoc patches for cross building.
 
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Mark Brand <mabrand@mabrand.nl>
-Date: Fri, 24 Sep 2010 23:31:24 +0200
-Subject: [PATCH 1/3] s,DllMain,static _disabled_DllMain,
+From: v1993 <v19930312@gmail.com>
+Date: Mon, 6 Dec 2021 21:38:36 +0300
+Subject: [PATCH 1/1] Fix static builds for Windows
 
 
 diff --git a/gdk-pixbuf/gdk-pixbuf-io.c b/gdk-pixbuf/gdk-pixbuf-io.c
 index 1111111..2222222 100644
 --- a/gdk-pixbuf/gdk-pixbuf-io.c
 +++ b/gdk-pixbuf/gdk-pixbuf-io.c
-@@ -193,12 +193,12 @@ get_file_formats (void)
+@@ -186,8 +186,9 @@ get_file_formats (void)
  
- #ifdef G_OS_WIN32
+ /* DllMain function needed to tuck away the gdk-pixbuf DLL handle */
  
--/* DllMain function needed to tuck away the gdk-pixbuf DLL handle */
-+/* static _disabled_DllMain function needed to tuck away the gdk-pixbuf DLL handle */
- 
- static HMODULE gdk_pixbuf_dll;
+-static HMODULE gdk_pixbuf_dll;
++static HMODULE gdk_pixbuf_dll = NULL;
```

Comment:
FYI

During the upstream reviewing with gdk-pixbuf they mentioned that the static variable automatically get initialized by the compiler to null if it's a pointer type, which is true. Meaning an explicit = NULL is not required. 

I argued that initializing the static variable isn't wrong.. but anyway.

Reply:
I think I'll keep this for now in MXE's version. If patch gets into upstream, with or without this change, we'll be able to drop this entirely.